### PR TITLE
Fixes issue detail and contacts in more issues

### PR DESCRIPTION
**Pull request by nickoneill**
- Storyboard connections for getting from the more issues list -> issue detail weren't set
- Both main and moreissues/inactive issue endpoint needs a location sent to get contacts back

### CODE REVIEW

File: FiveCalls/FiveCalls/FetchIssuesOperation.swift [hunk 1]
```diff
@@ -43,11 +34,18 @@ class FetchIssuesOperation : BaseOperation {
         urlComponents.scheme = "https"
         urlComponents.host = "5calls.org"
         urlComponents.path = "/issues/"
-        
-        if let queryString = query.queryString() {
-            urlComponents.query = queryString
+
+        var queryItems: [URLQueryItem] = []
+        if fetchInactive {
+            let inactiveQueryItem = URLQueryItem(name: "inactive", value: "true")
```
Comment (subdigital) [inline FiveCalls/FiveCalls/FetchIssuesOperation.swift]: TIL...

File: FiveCalls/FiveCalls/IssueDetailViewController.swift [hunk 2]
```diff
@@ -196,7 +196,7 @@ extension IssueDetailViewController : EditLocationViewControllerDelegate {
     }
     
     func editLocationViewController(_ vc: EditLocationViewController, didUpdateLocation location: UserLocation) {
-        issuesManager.fetchIssues(.top(location)) { result in
+        issuesManager.fetchIssues(false, location: location) { result in
```
Comment (subdigital) [inline FiveCalls/FiveCalls/IssueDetailViewController.swift]: I think I like the enum style better, but I see the need to extract the location and pass that regardless.  Maybe just remove the associated type and use the enum like this?

```swift
fetchIssues(.all, location: location)  // or .top
```

File: FiveCalls/FiveCalls/IssuesViewController.swift [hunk 3]
```diff
@@ -105,8 +105,8 @@ class IssuesViewController : UITableViewController {
         isLoading = true
         tableView.reloadEmptyDataSet()
         issuesDelegate?.didStartLoadingIssues()
-        let query: IssuesManager.Query = shouldFetchAllIssues ? .all : .top(UserLocation.current)
```
Comment (subdigital) [inline FiveCalls/FiveCalls/IssuesViewController.swift]: and this line could probably have been improved as a separate method that returns the appropriate `Query` case.

**Comment by subdigital** [comment]
Note for @ianyh, you probably want to rebase this branch once Nick is done.

**Comment by ianyh** [comment]
Okay. 👍

File: FiveCalls/FiveCalls/IssueDetailViewController.swift (hunk 2 above)
Comment (nickoneill) [inline FiveCalls/FiveCalls/IssueDetailViewController.swift]: My first attempt was just both cases of `Query` had the `UserLocation` associated type. Was easy to swap to that, but it feels funky. I'll go back to the plain enum here.

File: FiveCalls/FiveCalls/IssuesViewController.swift (hunk 3 above)
Comment (nickoneill) [inline FiveCalls/FiveCalls/IssuesViewController.swift]: Based on this `IBInspectable` thing? Not sure how that would work

**Comment by nickoneill** [comment]
OK, this should be set with the requested changes.

**Comment by nickoneill** [comment]
ping @subdigital

**Pull request merged**